src/ui/markdown.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var Util = require('../util');
+
+var g = Util.getGlobal(),
+    _t = Util.gettext;
+
+function createMarkdownPlugin () {
+    var converter = null;
+
+    if (typeof g.Showdown !== "undefined" &&
+        g.Showdown !== null &&
+        typeof g.Showdown.converter !== "undefined" &&
+        g.Showdown.converter !== null) {
+        converter = new g.Showdown.converter();
```

Comment:
Just out of curiosity, is there a case where Showdown can be defined without a `converter` method? Is this a versioning thing?